tests/input/c/negative_value.c

```diff
@@ -1,10 +1,10 @@
 #include <stdio.h>
 
 int main(void) {
-	int i = -1;
-	int j = 30 * -1;
+    int i = -1;
+    int j = 30 * -1;
 
-	int frames = 127;
-	frames = ((frames + 15) & -16);
-	return 0;
-}
\ No newline at end of file
+    int frames = 127;
+    frames = ((frames + 15) & - 16);
+    return 0;
+}
```

Comment:
This change breaks the test as it adds a space after the '-'. 
The test is supposed to test the behavior of a minus that is attached to a digit (CT_NEG vs. CT_ARITH)